web/src/routes/OrganizationsList/Organization/Tabs/TeamsAndMembership/TeamsView/ManageMembers/ManageMembersList.tsx

```diff
@@ -243,7 +243,7 @@ export default function ManageMembersList(props: ManageMembersListProps) {
               data-testid={TableModeType.Invited}
               isSelected={tableMode == TableModeType.Invited}
               onChange={onTableModeChange}
-              isDisabled={!organization.is_admin}
+              isDisabled={!organization.is_admin || teamSyncInfo != null}
```

Comment:
nit: this line could use `!==` for strict inequality

Reply:
how about checking for both explicitly? I'd avoid non-strict equality for clarity and to prevent potential bugs.

Reply:
Hmm,sure I can. But, I am curious what kind of bugs are you referring to?